src/lib/Tests/Form/DataTransformer/UDWBasedValueViewTransformerTest.php

```diff
@@ -84,29 +84,28 @@ public function dataProviderForReverseTransform(): array
         ];
     }
 
-    /**
-     * @dataProvider dataProviderForReverseTransformThrowsTransformationFailedException
-     */
-    public function testReverseTransformThrowsTransformationFailedException(string $exceptionClass)
+    public function testTransformWithDeletedLocation(): void
     {
-        $this->expectException(TransformationFailedException::class);
-
         $this->locationService
-            ->expects($this->any())
             ->method('loadLocation')
             ->willThrowException(
-                $this->createMock($exceptionClass)
+                $this->createMock(NotFoundException::class)
             );
 
-        $this->transformer->reverseTransform('54,56,58');
+        self::assertEmpty($this->transformer->transform(['/1/2/54']));
     }
 
-    public function dataProviderForReverseTransformThrowsTransformationFailedException(): array
+    public function testReverseTransformThrowsTransformationFailedException(): void
     {
-        return [
-            [NotFoundException::class],
-            [UnauthorizedException::class],
-        ];
+        $this->expectException(TransformationFailedException::class);
+
+        $this->locationService
+            ->method('loadLocation')
+            ->willThrowException(
+                $this->createMock(UnauthorizedException::class)
```

Comment:
Inconsistent static / `$this` usage. Keep one or the other I guess?

I mean, if you want to make Andrew extra happy :heart_eyes: then use static variant :stuck_out_tongue_winking_eye:

Reply:
> Inconsistent static / `$this` usage. Keep one or the other I guess?
> 
> I mean, if you want to make Andrew extra happy  then use static variant

AFAICS createMock is not static :stuck_out_tongue_winking_eye:

Reply:
Darn, you got me.

_And I would get away with it, were it not for those meddling kids..._


Disregard :smile: